src/components/Journey/Journey.tsx

```diff
@@ -30,7 +30,9 @@ const Journey:React.FC = () => {
         </div>
 
         <div className="btn-container">
+          <Link to="/survey">
           <button className="btn">Start</button>
+          </Link>
```

Comment:
I was worried that the Link tag would mess up with the styling but I am glad it didn't